package.json

```diff
@@ -15,8 +15,10 @@
     "lint": "run-p lint:src lint:test",
     "test": "run-s lint test:unit",
     "integration": "run-s build test:integration",
+    "integration:emulator": "run-s build test:integration:emulator",
     "test:unit": "mocha test/unit/*.spec.ts --require ts-node/register",
     "test:integration": "mocha test/integration/*.ts --slow 5000 --timeout 20000 --require ts-node/register",
+    "test:integration:emulator": "firebase emulators:exec --project fake-project-id --only auth,database,firestore 'mocha \"test/integration/{auth,database,firestore}.spec.ts\" --slow 5000 --timeout 20000 --require ts-node/register'",
```

Comment:
Let's do this in the workflow configuration. Ideally we shouldn't add any emulator specific commands to this file.

Reply:
Done.